src/build.py

```diff
@@ -195,6 +195,10 @@ def NodePlatformName():
     os.path.join(BINARYEN_SRC_DIR, 'test',
                  's2wasm_known_binaryen_shell_test_failures.txt')]
 
+# Optimization levels
+BARE_TEST_OPT_FLAGS = ['O0', 'O2']
+BINARYEN_TEST_OPT_FLAGS = ['O0', 'O3']
```

Comment:
Let's call this `EMSCRIPTEN_TEST_OPT_FLAGS` since the bare tests also use pieces of Binaryen. (And Binaryen is really just an implementation detail of emscripten).